### ISSUE TITLE
[CoE Starter Kit - Feature]: DLP Editor V2 to support policies with single environment scope

### ISSUE THREAD

**Issue by annsarP** [microsoft/coe-starter-kit#6356]
### Is your feature request related to a problem? Please describe.

Hi! While using DLP Editor V2 I found out that this app is not supporting DLP with single environment scope (policies created by environment admins). 
![image](https://github.com/microsoft/coe-starter-kit/assets/127112137/1e3e80be-b15c-45e6-b665-e5c0545c514e)

We can't create a copy (which of course make sense since tenant-level admins can create tenant-level dlp policies): 
![image](https://github.com/microsoft/coe-starter-kit/assets/127112137/b36541ab-d1f1-4e48-8129-ac0b7511413e)

**or see which environment (1) is assigned to this policy:**
![image](https://github.com/microsoft/coe-starter-kit/assets/127112137/f82f7d41-c82b-4c61-b924-d4f710abb9da)

### Describe the solution you'd like

Would it be possible to: 

- **add information which environment is assigned to policy in details screen**, 
- create copy of environment-level policy on environment or tenant-level?

**Comment by Jenefer-Monroe** [comment]
Can you please try in the DLP Impact Analysis [Preview] app? It is the replacement of the former.

**Comment by annsarP** [comment]
Hi! I tested DLP Impact Analysis [Preview] app, and I faced following issues:

- When I  select single environment DLP Policy I can go trough steps by clicking "Next" button but there's no "Scope" on the list of the steps (I can reach it using "Next" button, so it's available, but just not listed): 
![image](https://github.com/microsoft/coe-starter-kit/assets/127112137/9d7960cd-20a4-4ab1-ad1f-62bf83dfc0ac)
same for the list of environments:
![image](https://github.com/microsoft/coe-starter-kit/assets/127112137/d2407f25-42de-42f8-ba81-797fd34f5631)

- When we go to "Review" step app shows 0 environments selected, even though as you can see on the previous screen, 1 environment has been added to the policy:
![image](https://github.com/microsoft/coe-starter-kit/assets/127112137/d6ffa976-484d-4a35-bb9d-6cce0cef39cd)

- If I try to copy environment I got notification that it has been created, while it wasn't. If it's not possible for PP Admin to create copy of this kind of environment it would be good to either disable Copy button for single environment DLP policies or to show error message:
![image](https://github.com/microsoft/coe-starter-kit/assets/127112137/0919ad1c-62fb-4a1e-9eb0-d2bfac68ea54)

I know the app is in the Preview stage but I thought it would be helpful to share this feedback. 😊

**Comment by Jenefer-Monroe** [comment]
Thank you for the update! Sending over to the area owner.

**Comment by manuelap-msft** [comment]
That's unfortunately a limitation of the Power Platform for Admin connector, it only works for tenant level policies not environment level policies.

**Comment by Jenefer-Monroe** [comment]
closing out as no further action for starter kit team

**Comment by annsarP** [comment]
> That's unfortunately a limitation of the Power Platform for Admin connector, it only works for tenant level policies not environment level policies.
> 
> PP Admins are also not able to create environment level policies in the Admin Center, that's an action only available to Environment Admins. The workaround is to create a tenant level policy that's only assigned to one environment.

I see. So would there be possibility to add requested blockers or infos (errors) in app while trying to manage this kind of environments? I think currently it might be confused for admins if they receive information that copy was created while it wasn't :/

**Comment by manuelap-msft** [comment]
Yes, we can look into adding an info banner / warning. Putting on our feature backlog.

**Comment by harini-2-y** [comment]
This issue is being closed as its scope is now included in a newly created consolidated enhancement request for centralized management of orphaned components (Apps, Flows, Connection References) in the CoE Starter Kit.

New Issue Reference: [[CoE Starter Kit - Feature]: https://github.com/https://github.com/microsoft/coe-starter-kit/issues/10339

Thank you for your contributions
